src/components/campaign-wizard/CampaignWizard.jsx

```diff
@@ -132,8 +132,8 @@ dataset and all the metadata associated with this vaccination campaign.`),
         }
     });
 
-    onStepChangeRequest = currentStep => {
-        return getValidationMessages(this.state.campaign, currentStep.validationKeys);
+    onStepChangeRequest = async currentStep => {
+        return await getValidationMessages(this.state.campaign, currentStep.validationKeys);
```

Comment:
Note: It's not necessary to add async/await in this case, but some programmers prefer it to be explicit, I guess that's your case. It's ok as it is.